__tests__/app.test.js

```diff
@@ -168,3 +138,48 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/2/comments", () => {
+  test("GET - status : 200 - responds with all properties", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        response.body.comments.forEach((comment) => {
```

Comment:
If you're going to use a forEach, we should be asserting the length of the comment array